### PR TITLE
Fill in the Windows file information (version, name, copyright, year).

### PR DESCRIPTION
**Pull request by AlexandraTrifan**
Fill in the Windows file information (version, name, copyright, year).  Requested in issue #94.

Signed-off-by: Alexandra Trifan <Alexandra.Trifan@analog.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -4,6 +4,9 @@ project(libiio C)
 set(LIBIIO_VERSION_MAJOR 0)
 set(LIBIIO_VERSION_MINOR 12)
 set(VERSION "${LIBIIO_VERSION_MAJOR}.${LIBIIO_VERSION_MINOR}")
+if (WIN32)
+	string(TIMESTAMP BUILD_YEAR "%Y")
+endif()
```
Comment (commodo) [inline CMakeLists.txt]: curios: do we need this for non-Win32 builds ? or just for Win32 ?

**Comment by commodo** [comment]
rest looks fine from me

[LGTM](http://cdn.343i.org/forum/fileshare/gallery/album_615/gallery_77261_615_18114.jpg)

File: CMakeLists.txt (hunk 1 above)
Comment (pcercuei) [reply CMakeLists.txt]: The WIN32 variable is defined on Windows, Cygwin and MinGW, independently of the CPU architecture (stupid, I know).

**Comment by AlexandraTrifan** [comment]
Done. Sorry for the delay.

**Comment by pcercuei** [comment]
Unfortunately the AppVeyor build seems broken (for a different reason). The appveyor.yml does things that are not needed anymore (like forcing a re-install of GCC to get the latest version). Could you maybe take a look?

**Comment by AlexandraTrifan** [comment]
I added a commit regarding this. In my opinion, the initial core update can be dropped, since we update the package list before installing certain packages. 
If this option does not look ok to you, it seems that there is another workaround using an undocumented option _--ask_ (mainly added for internal usage, though).

**Comment by pcercuei** [comment]
That's OK to me, we can merge. Thanks!

**Pull request merged**